src/Normalizer/Normalizer.php

```diff
@@ -130,7 +131,11 @@ private function getFieldsByFieldNormalizationMappings(
     ): array {
         $data = [];
         foreach ($normalizationFieldMappings as $normalizationFieldMapping) {
-            if (!$this->isWithinGroup($context, $normalizationFieldMapping)) {
+            if (true !== $this->isCompliant($context, $normalizationFieldMapping, $object)) {
```

Comment:
`if (true !== $this->isCompliant($context, $normalizationFieldMapping, $object)) {`

=>

`if (!$this->isCompliant($context, $normalizationFieldMapping, $object)) {`

type checks here drop the readability for no gain

Reply:
i got used to it because of our team and the "pre return type" times

Reply:
all 5 occurrences changed